app/templates/ui/app/error/errorInterceptor.service.js

```diff
@@ -8,24 +8,43 @@
     }]);
 
   ErrorInterceptor.$inject = ['$q', '$injector', 'ngToast'];
+
   function ErrorInterceptor($q, $injector, toast) {
     var errorInterceptor = {
       responseError: function(rejection) {
+        var msg;
+        var toastMsg;
         if (rejection.status === 500 || rejection.status === 400) {
           var messageBoardService = $injector.get('messageBoardService');
-          var msg;
           if (rejection.data && rejection.data.errorResponse) {
             msg = {
               title: rejection.data.errorResponse.status,
               body: rejection.data.errorResponse.message
             };
           } else {
             msg = {
-              title: (rejection.status === 400) ? 'Bad Request' : 'Internal Server Error'
+              title: (rejection.status === 400) ? 'Bad Request' : 'Internal Server Error',
+              body: ''
             };
           }
           messageBoardService.message(msg);
-          var toastMsg = '<strong>' + msg.title + '</strong><p>' + msg.body + '</p>';
+          toastMsg = '<strong>' + msg.title + '</strong><p>' + msg.body + '</p>';
+          toast.danger(toastMsg);
+        } else if (rejection.status === 401) {
+          if (rejection.data && rejection.data.errorResponse) {
+            msg = {
+              title: rejection.data.errorResponse.status,
+              body: rejection.data.errorResponse.message
+            };
+          } else {
+            msg = {
+              title: (rejection.status === 401) ? 'Unauthorised' : 'Internal Server Error',
+              body: ''
+            };
+          }
```

Comment:
Regarding the entire if above, isn't it always title Unauthorized? You only enter this branch if rejection.status === 401..